docs/usage-wharfyml/the-wharf-ci-yml-file.md

```diff
@@ -5,7 +5,7 @@
 <!-- div:left-panel -->
 
 The `.wharf-ci.yml` file is located at the **root** path of each projects
-repositories. The file is sometimes referred to as the projects
+repositories. The file is sometimes referred to as the project's
```

Comment:
I always mess this up. Doesn't "project's" mean "project is"? I've never understood this

Reply:
Nope, it describes posession :) 
https://en.wikipedia.org/wiki/English_possessive#Nouns_and_noun_phrases

Reply:
I just pushed a few more grammar fixes

Reply:
> The possessive /.../ is made by suffixing a morpheme which is represented orthographically as 's

*Uuuuuh sure*

Jokes aside, that wiki page explained so much I didn't grasp. Thanks for that! Never knew about the "Formation of possessive construction" concept before.